agent/bench-scripts/pbench-uperf

```diff
@@ -182,17 +183,17 @@ function stop_server {
 	local server=$1
 	local server_port=$2
 	local message=$3
-	local uperf_pid=`ssh $ssh_opts $server netstat -tlnp | grep ":$server_port " | awk '{print $7}' | awk -F/ '{print $1}'`
-	if [ ! -z "$uperf_pid" ]; then
+	local uperf_pids=`ssh $ssh_opts $server ss -tlnp --vsock | grep ":$server_port " | awk '{print $7}' | grep -o -e 'pid=[0-9]+' | cut -c 5-`
+	if [ ! -z "$uperf_pids" ]; then
 		if [ $message -eq 1 ]; then
-			echo found uperf pid $uperf_pid on $server:$server_port, killing
+			echo "found uperf pid(s) $uperf_pids on $server:$server_port, killing"
 		fi
-		ssh $ssh_opts $server kill $uperf_pid
+        ssh $ssh_opts $server "for pid in $uperf_pids; do kill $uperf_pid; done"
```

Comment:
Shouldn't the argument to `kill` here be `\$pid` instead of `$uperf_pid`?

Reply:
yes, tested that before adding the loop, should have tested it again afterwards...

Reply:
There is also a tabs vs spaces problem on line 191.